openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.test.tsx

```diff
@@ -115,4 +119,40 @@ describe('Test FeedList Component', () => {
     expect(feed1).toBeInTheDocument();
     expect(feed2).toBeInTheDocument();
   });
+
+  it('No data placeholders should not displayed if feedList is empty but feeds are loading', async () => {
+    const { container } = render(
+      <ActivityFeedList {...mockFeedListProp} isFeedLoading feedList={[]} />,
+      {
+        wrapper: MemoryRouter,
+      }
+    );
+
+    const noFeedPlaceholderContainer = queryByTestId(
+      container,
+      'no-data-placeholder-container'
+    );
+
+    expect(noFeedPlaceholderContainer).toBeNull();
```

Comment:
Do you think we can check for the loader here as well?

Reply:
Actually, we do not show the loader in this Component. The loaders are shown outside the component where it is used.